### DISCUSSION TITLE
구조분해할당 어려워요😢

### DISCUSSION THREAD

**Discussion by pyunji** [pyunji/Hyundai-ITnE-Team_QuadCore#19] [Q&A]
관련된 좋은 자료를 알고 계신 분은 공유해주세요 !

**Comment by chaeyeon4u** [comment]
w3school이랑 비슷한 공식문서 사이트에요! ES5 버전 전까지 보시면 될 것 같아요!
[구조분해할당](https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment)

**Reply by pyunji** [reply]
감사합니다 👍👍

**Reply by hyk339** [reply]
감사합니다~!
